app/webpack/reactjs/components/spinner.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from "react";
+
+interface SpinnerProps {}
+
+export const Spinner: FC<SpinnerProps> = () => {
+  return (
```

Comment:
This is the same as the spinner in `application_helper.rb` but in react.  

We might want to move to using Spin.js but this seems to work pretty well.  It's not great for resizing but for now i think it'll do.   It's likely a bit more light weight than a js spinner.